### ISSUE TITLE
Perl warnings

### ISSUE THREAD

**Issue by bfontaine** [statico/cadubi#1]
Hello,
I updated the Homebrew formula to use this repo (https://github.com/Homebrew/homebrew/pull/43986); and using both v1.3 and the latest commit I get Perl warnings:

With the 1.3 release:

```
$ cadubi -h
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 347.
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 353.
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 354.
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 855.
defined(@array) is deprecated at /usr/local/bin/cadubi line 895.
    (Maybe you should just omit the defined()?)

usage: /usr/local/bin/cadubi [OPTIONS] [FILE]

Available options:
  -h, --help              what you're looking at now
  -m, --mute              turn off beeping
  -s, --size [W] [H]      sets the size of the console for use with
                          cadubi, where W is number of columns and H
                          is number of rows.
  -v, --version           show cadubi version
```

With the latest commit:

```
$ cadubi -h
Unquoted string "raw" may clash with future reserved word at /usr/local/bin/cadubi line 50.
Unquoted string "restore" may clash with future reserved word at /usr/local/bin/cadubi line 68.
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 341.
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 347.
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 348.
Bareword found in conditional at /usr/local/bin/cadubi line 441.
Bareword found in conditional at /usr/local/bin/cadubi line 738.
Using an array as a reference is deprecated at /usr/local/bin/cadubi line 842.
Useless use of a constant ("IN") in void context at /usr/local/bin/cadubi line 735.
defined(@array) is deprecated at /usr/local/bin/cadubi line 882.
    (Maybe you should just omit the defined()?)
Name "main::bug" used only once: possible typo at /usr/local/bin/cadubi line 818.
Name "main::controlkeys" used only once: possible typo at /usr/local/bin/cadubi line 448.
Name "main::oldmap" used only once: possible typo at /usr/local/bin/cadubi line 699.
Use of uninitialized value $option in concatenation (.) or string at /usr/local/bin/cadubi line 382.

usage: /usr/local/bin/cadubi [OPTIONS] [FILE]

Available options:
  -h, --help              what you're looking at now
  -m, --mute              turn off beeping
  -s, --size [W] [H]      sets the size of the console for use with
                          cadubi, where W is number of columns and H
                          is number of rows.
  -v, --version           show cadubi version
```

I’m using OS X 10.10.5.

```
$ perl --version

This is perl 5, version 18, subversion 2 (v5.18.2) built for darwin-thread-multi-2level
(with 2 registered patches, see perl -V for more detail)
…snip…
```

And the output of `perl -V`: https://gist.github.com/bfontaine/dde998c55f878991c19b

**Comment by statico** [comment]
Thanks @bfontaine! I'll try to get around to fixing these soon.

**Comment by statico** [comment]
Fixed in v1.3.1

**Comment by bfontaine** [comment]
Thanks!
